src/apis/platformvm/camino_executor.ts

```diff
@@ -58,6 +62,91 @@ export class CaminoExecutor {
     return
   }
 
+  /**
+   * Helper function which creates an unsigned [[CaminoAddValidatorTx]]. For more granular control, you may create your own
+   * [[UnsignedTx]] manually and import the [[CaminoAddValidatorTx]] class directly.
+   *
+   * @param networkID Networkid, [[DefaultNetworkID]]
+   * @param blockchainID Blockchainid, default undefined
+   * @param toAddresses An array of addresses as {@link https://github.com/feross/buffer|Buffer} who received the staked tokens at the end of the staking period
+   * @param fromAddresses An array of addresses as {@link https://github.com/feross/buffer|Buffer} who own the staking UTXOs the fees in AVAX
+   * @param changeAddresses An array of addresses as {@link https://github.com/feross/buffer|Buffer} who gets the change leftover from the fee payment
+   * @param nodeID The node ID of the validator being added.
+   * @param startTime The Unix time when the validator starts validating the Primary Network.
+   * @param endTime The Unix time when the validator stops validating the Primary Network (and staked AVAX is returned).
+   * @param stakeAmount The amount being delegated as a {@link https://github.com/indutny/bn.js/|BN}
+   * @param rewardAddresses The addresses which will recieve the rewards from the delegated stake.
+   * @param rewardLocktime Optional. The locktime field created in the resulting reward outputs
+   * @param rewardThreshold Opional. The number of signatures required to spend the funds in the resultant reward UTXO. Default 1.
+   * @param memo Optional contains arbitrary bytes, up to 256 bytes
+   *
+   * @returns An unsigned transaction created from the passed in parameters.
+   */
+  buildCaminoAddValidatorTx = async (
+    networkID: number = DefaultNetworkID,
+    blockchainID: Buffer,
+    to: Buffer[],
+    from: Buffer[],
+    change: Buffer[],
+    nodeID: Buffer,
+    startTime: BN,
+    endTime: BN,
+    stakeAmount: BN,
+    rewards: Buffer[],
+    rewardLocktime: BN = new BN(0),
+    rewardThreshold: number = 1,
+    memo: Buffer = undefined,
+    avaxAssetID: Buffer
+  ): Promise<UnsignedTx> => {
+    let ins: TransferableInput[] = []
+    let outs: TransferableOutput[] = []
+
+    const zero: BN = new BN(0)
+    const now: BN = UnixNow()
+    if (startTime.lt(now) || endTime.lte(startTime)) {
+      throw new TimeError(
+        "buildCaminoAddValidatorTx -- startTime must be in the future and endTime must come after startTime"
```

Comment:
`and endTime must come after startTime` isn't the assertion checking if endTime is after OR equal to startTime? (lte) I think it should be (lt), what do you think?

Reply:
It is checking if endtime is lower or equal startTime and then it's error